### PR TITLE
getMemberTypes() respects source order

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

**Comment by chriseth** [comment]
No change to tests?

**Comment by LefterisJP** [comment]
As I told you in person, all tests passed locally and this should be tested anyway by all our existing tests.

**Comment by gavofyork** [comment]
rebase?

File: libsolidity/Types.cpp [hunk 1]
```diff
@@ -566,18 +566,18 @@ MemberList const& ContractType::getMembers() const
 	if (!m_members)
 	{
 		// All address members and all interface functions
-		map<string, TypePointer> members(IntegerType::AddressMemberList.begin(),
-										 IntegerType::AddressMemberList.end());
+		vector<pair<string, TypePointer>> members(IntegerType::AddressMemberList.begin(),
```
Comment (gavofyork) [inline libsolidity/Types.cpp]: not multimap?

**Comment by LefterisJP** [comment]
rebased. @gavofyork vector of pairs seems to work just fine. A multimap may have worked well enough too though.

**Pull request merged**